client/landing/stepper/declarative-flow/internals/steps-repository/hundred-year-plan-step-wrapper/info-modal.tsx

```diff
@@ -155,6 +155,54 @@ export default function InfoModal( {
 		flowName === HUNDRED_YEAR_DOMAIN_FLOW
 			? translate( '100-Year Domain' )
 			: getPlan( PLAN_100_YEARS )?.getTitle();
+	const featureOneTitle =
```

Comment:
This works and is probably the only way under the current structure. However, it would be better if `info-modal` simply shows the copies assigned to it, rather than conditionally showing different copies based on the flow name. 

Decoupling this is non-trivial though. 

* We'd need to extract the mechanism of customizing step wrapper component: https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/domains/index.tsx#L305
* We'd also need to implement the mechanism of passing . In this case, conceptually only `hundred-year-plan` and `hundred-year-domain` flows know what to show in the info modal, so those should be configurable right in their flow implementation, e.g. https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/hundred-year-domain.ts#L65 . 

@alshakero Are there existing ways that we can better structure this?

Reply:
Thanks for taking a look @southp. Should we handle restructuring this code in a separate PR? Can we go ahead and merge this?